Sequencer.cs

```diff
@@ -16,39 +16,99 @@ namespace WordSequence
 {
     public class Sequencer : CustomPwGenerator
     {
-        private string GetConfigurationPath()
+        /* if returnEmptyConfig is true, or a user config file is found, returns
+         * the path to the user config file.
+         *
+         * if returnEmptyConfig is false, or the user config file path is not
+         * specified, and the user config file is not found, returns the path to
+         * the global config.

```

Comment:
I don't think the `returnEmptyConfig` is relevant, unless I don't understand your comment.  The way I see it, if a `userConfigPath` config exists and the file actually exists, we return it, otherwise, we return the `configPath` if it exists, otherwise null.